Project/Assets/Editor/Twine Importer/TwineJsonParser.cs

```diff
@@ -23,14 +28,85 @@ public class TwineJsonParser {
 	public static void ReadJson (string jsonString) {
 		JSONNode parsedJson = JSON.Parse(jsonString);
 		JSONArray parsedArray = parsedJson.AsArray;
-
-		// Demo code, printing the ID and name of each node:
+		GameObject[] twineNodes = new GameObject[parsedArray.Count];
+		
+		int count = 0;
+		// create parent game object
+		GameObject parent = new GameObject("Story");
+		// make GameObject nodes out of every twine/json node
 		foreach (JSONNode storyNode in parsedArray) {
-			Debug.Log ("Node ID: " + storyNode[NodeIdKey]);
-			Debug.Log ("Node name: " + storyNode[NodeNameKey]);
+			GameObject twineNode = MakeGameObjectFromStory (parent, storyNode);
+			twineNodes [count] = twineNode;
+			++count;
+		}
+		// make dictionary
+		Dictionary<string, GameObject> objDict = new Dictionary<string, GameObject>();
+		// add things to dictionary
+		foreach (GameObject node in twineNodes) {
+			objDict.Add (node.name, node);
 		}
+		FindChildren (twineNodes, objDict);
+		// create the large prefab, and kill the GameObject that is not a prefab
+		PrefabUtility.CreatePrefab ("Assets/Ignored/Story.prefab", parent);
+		GameObject.DestroyImmediate (parent);
+	}
 
+	// Finds the children of each of the gameObjects from the other gameObjects
+	// Iterates through the list of nodes, gets an array of children, and assigns
+	// gameObject children to the node that match the names of the array of children
+	public static void FindChildren (GameObject[] nodes, Dictionary<string, GameObject> objDict) {
```

Comment:
Another nitpicky naming thing, but does this function also *match* the children or store the relationships somehow? Maybe could we name it `MatchChildren` since `Find` kinda makes it seem like it's just returning the Relationships for us to do something with, whereas `Match` makes it seem like the function itself is acting on the children rather than just returning something for us to use.